src/main/java/pages/PhonesPage.java

```diff
@@ -0,0 +1,42 @@
+package pages;
+
+import base.PageBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhonesPage extends PageBase {
+    public PhonesPage(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final By sortList = By.id("input-sort");
+    public List<String> ExpectedAlphabeticalOrder;
+    public List<String> ActualAlphabeticalOrder;
+
+
+    public void sortByName() {
+        ExpectedAlphabeticalOrder = new ArrayList<>();
+        ExpectedAlphabeticalOrder.add("HTC Touch HD");
+        ExpectedAlphabeticalOrder.add("iPhone");
+        ExpectedAlphabeticalOrder.add("Palm Treo Pro");
+        System.out.println("Expected listing: "+ExpectedAlphabeticalOrder);
```

Comment:
try to user Reporter.Log instead of System.out.println, as when you do this, the message will be displayed in allure report as well